model/qti/container/Container.php

```diff
@@ -222,6 +223,19 @@ public function fixNonvoidTags($html)
             // correctly closed element
             return trim(mb_substr($matches[0], 0, -2), 'UTF-8') . '></' . $matches[2] . '>';
         }, $html);
+
+        $pregLastError = preg_last_error();
```

Comment:
Why did you remove the private method?

Reply:
Extracting this method with argument type hint cause issues because `$content` can be string, array or null. Extracting this business logic to separate method was useless and less readable.